content/applications/hr/attendances/kiosks.rst

```diff
@@ -0,0 +1,181 @@
+======
+Kiosks
+======
+
+Odoo's *Attendances* app allows employees to check in and out of work either directly from the
+databse, or from a kiosk. Since only employee's who have access to the database (referred to as
```

Comment:
```suggestion
database, or from a kiosk. Since only employees who have access to the database (referred to as
```
Typo and changed employees from possessive to plural